include/nds/arm9/math.h

```diff
@@ -214,7 +230,6 @@ static inline void div64_asynch(int64_t num, int32_t den)
 static inline int32_t div64_result(void)
 {
     while (REG_DIVCNT & DIV_BUSY);
-
```

Comment:
Don't change whitespace randomly in other parts of the code, it makes it harder to analyze the history of the file.